File: trainAlgorithms/src/predictionValue.py
from typing import NamedTuple, List

import numpy as np
from numpy import float64


class PredictionValue(NamedTuple):
    date: List[str]
    dt: List[float64]
    rf: List[float64]
    mlp: List[float64]
    real: List[float64]
    rmse_dt: List[float64]
    mape_dt: List[float64]
    mse_dt: List[float64]
    rmse_rf: List[float64]
    mape_rf: List[float64]
    mse_rf: List[float64]
    rmse_mlp: List[float64]
    mape_mlp: List[float64]
    mse_mlp: List[float64]

File: PredictionManagement/src/main/java/com/utcn/ds/predictionmanagement/controller/dto/PredictionValue.java
package com.utcn.ds.predictionmanagement.controller.dto;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PredictionValue {

    private List<String> date;
    private List<String> dt;
    private List<String> rf;
    private List<String> mlp;
    private List<String> real;
    private List<String> rmse_dt;
    private List<String> mape_dt;
    private List<String> mse_dt;
    private List<String> rmse_rf;
    private List<String> mape_rf;
    private List<String> mse_rf;
    private List<String> rmse_mlp;
    private List<String> mape_mlp;
    private List<String> mse_mlp;
}

